### ISSUE TITLE
Feature request: add Loudness Equalization toggle

### ISSUE THREAD

**Issue by SamiPirbay** [File-New-Project/EarTrumpet#191]
Loudness Equalization can be found in Speakers Properties &rightarrow; Enhancements. It's a bit of a trek to get there.

It's useful when watching Netflix if you don't want action sequences to be louder than the rest of the movie, for instance.

**Comment by riverar** [comment]
Sounds like a cool idea; will investigate exposing enhancements

**Comment by Riajyuu** [comment]
Just a quick note: some laptop doesn't have this in system settings.

**Comment by fsfod** [comment]
I managed to get this working in my branch here https://github.com/fsfod/EarTrumpet/commits/loudness_fx i added it checkout in the right click menu that changes it for the default device. Tthe tldr of it is setting a FX config option with IPolicyConfigWin7.SetPropertyValue with the guid of E0A941A0-88A2-4df5-8D6B-DD20BB06E8FB and property id of 4

**Comment by SamiPirbay** [comment]
It doesn't work on my computers. I believe this GUID is specific to Realtek High Definition Audio devices.

It's still useful since Realtek is in so many systems, but I think the UI should somehow warn the user or disable the checkbox in unsupported systems

**Comment by fsfod** [comment]
You could maybe find the FX property store GUID for your device using some registry monitor and toggling the option in the windows property page just filter to registry path containing "FxProperties". My code could just be toggling it on the wrong output device you idk.

**Comment by SamiPirbay** [comment]
I did a three-way registry diff (original state, state after toggle, and state after toggle back) and found that I can do it by changing the key `{fc52a749-4be9-4510-896e-966ba6525980},3` from `0b 00 e9 7e 01 00 00 00 - 00 00 00 00` to `0b 00 71 7f 01 00 00 00 - ff ff 00 00`. More obscure than a 0 to 1 switch but it works. I googled that key and found that it's called "MFPKEY_CORR_LOUDNESS_EQUALIZATION_ON".

So I can adapt your code for my computer, which is awesome, thanks :)

But how to make theses changes safe and generic enough for the release version?
